WordPress/src/main/java/org/wordpress/android/util/config/MySiteImprovementsFeatureConfig.kt

```diff
@@ -30,7 +30,7 @@ class MySiteImprovementsFeatureConfig
 
     // Temporary solution until the Firebase bug gets fixed
     override fun isEnabled(): Boolean {
-        return initEnabledValue(accountStore.accessToken.hashCode(), accountStore.hasAccessToken())
+        return initEnabledValue(accountStore.accessToken.orEmpty().hashCode(), accountStore.hasAccessToken())
```

Comment:
I verified that this fixes the crash and `accountStore.accessToken.orEmpty().hashCode()` returns `0` just as `accountStore.accessToken.hashCode()` does on a fresh install.

Due to this PR's urgency, I am going to approve it. However, I don't like that we are converting a `null` value to `0`, so I strongly suggest re-evaluating this approach and opening another PR against `develop`. A possible approach might be to use `accountStore.accessToken?.hashCode()` and making the `id` argument nullable.

I'll leave the rest to @renanferrari & @planarvoid.